### DISCUSSION TITLE
<title>Watch--> Custom --> Releases

### DISCUSSION THREAD

**Discussion by WhiteSymmetry** [community/community#40252] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Hi GitHub,

Watch--> Custom --> Releases 

Since I follow too many topics/versions/Releases, I receive more e-mails daily. Is it possible to separate the versions/Releases section as alpha, beta, RC and major versions/Releases? In this way, only the e-mails of those I want to follow come to me.

Best Regards,

**Comment by ossanna16** [comment]
Hi there @WhiteSymmetry and welcome to our community! Thank you for the feedback!
